core/src/main/java/org/apache/accumulo/core/metadata/schema/RootTabletMetadata.java

```diff
@@ -59,11 +58,7 @@ public class RootTabletMetadata {
         || fam.equals(FutureLocationColumnFamily.STR_NAME);
   };
 
-  // JSON Mapping Version 1. Released with Accumulo version 2.1.0
-  private static final int VERSION_1 = 1;
-  // JSON Mapping Version 2. Released with Accumulo version 3,1
-  private static final int VERSION_2 = 2;
-  private static final int VERSION = VERSION_2;
+  private static final int VERSION = 1;
```

Comment:
There may be value in having a version 2 for 3.1 changes.  The metadata encoding is distinct between 2.1 and 3.1 - in future versions, if someone skipped an upgrade version the distinct versions could help stopping the upgrade before irreversible changes could be made.

Reply:
>  if someone skipped an upgrade version the distinct versions could help stopping the upgrade 

We have higher level code that has the purpose of preventing that at an accumulo instance level.   

Thinking the version here in RootTabletMetadata may be more concerned with the encoding of its data and the structure of its json, none of which changed.   The fields names in the json and the structure of the json has not changed.

Reply:
> The inclusion of range for fencing is a change.

Yeah its a change in the data stored in the json.  In general the RootTabletMetadata object stores accumulo Key/Values in  json, but it does not do a lot to validate those key values are correct tablet metadata.  It relies on other shared code that validates all tablet metadata like the metadata constraint and tablet metadata parsing code that executes when reading any tablet metadata stored in different locations.   All of this other code can check for anomalies in the root tablet metadata, thinking its better to rely on that and not one off validations for just the root tablet metadata.

Reply:
I'll defer to you - but then why have a version number at all?  If we keep the version number, then the validation could occur at a higher level - but that implies that it will change.

Maybe the higher level checks (when needed) accept a range, so maybe the 3.1 changes would be a 1.1?  Higher level code the act on major or if necessary minor version changes.

My position is that if we are going to have a version number, then we should leverage that number to signal changes.  The serialization of files without fence ranges are not compatible with files with ranges.  The storage of those then end up different.  A meaningful version scheme could help signal the difference.